WalletWasabi/JsonConverters/Uint256JsonConverter.cs

```diff
@@ -4,25 +4,19 @@
 
 namespace WalletWasabi.JsonConverters;
 
-public class Uint256JsonConverter : JsonConverter
+public class Uint256JsonConverter : JsonConverter<uint256>
 {
 	/// <inheritdoc />
-	public override bool CanConvert(Type objectType)
+	public override uint256? ReadJson(JsonReader reader, Type objectType, uint256? existingValue, bool hasExistingValue, JsonSerializer serializer)
```

Comment:
`string.IsNullOrWhiteSpace` is actually not for free, it is O(n) where n is string length. It should be:

```cs
	/// <inheritdoc />
	public override uint256? ReadJson(JsonReader reader, Type objectType, uint256? existingValue, bool hasExistingValue, JsonSerializer serializer)
	{
		string value = Guard.Correct((string?)reader.Value);
		return value == "" ? default : new uint256(value);
	}
```

More courageous would be:

```cs
	/// <inheritdoc />
	public override uint256? ReadJson(JsonReader reader, Type objectType, uint256? existingValue, bool hasExistingValue, JsonSerializer serializer)
	{
		string? value = (string?)reader.Value;
		return value is null ? default : new uint256(value);
	}
```

as that checking for whitespace feels proactive but `new uint256(value)` will fail anyway if the format of that string is not correct. Also `new uint256(value)` contains a lenght check and it trims too.